test/callable-instance.js

```diff
@@ -51,9 +56,12 @@ test('callable-instance', async function (t) {
     const instance = new Es5Class()
     assert.strictEqual(instance.foo, 42)
 
+    /** @type {number} */
     instance.bar = 100
 
     // Instance is callable
+    /** @type {Es5Class} */
+    // @ts-ignore
```

Comment:
Could we make this `// @ts-expect-error` with a comment on why it is expected?

Reply:
Sure can